mappings/net/minecraft/util/DyeColor.mapping

```diff
@@ -38,14 +42,29 @@ CLASS net/minecraft/class_2674 net/minecraft/util/DyeColor
 		ARG 6 translationKey
 		ARG 7 color
 		ARG 8 formatting
+	METHOD <init> (Ljava/lang/String;IILjava/lang/String;ILnet/minecraft/class_65;I)V
+		ARG 3 id
+		ARG 4 translationKey
+		ARG 5 color
+		ARG 6 materialColor
+		ARG 7 idSwapped
 	METHOD method_11274 getId ()I
 	METHOD method_11275 getById (I)Lnet/minecraft/class_2674;
 		ARG 0 id
 	METHOD method_11276 getSwappedId ()I
-	METHOD method_11277 getColorById (I)Lnet/minecraft/class_2674;
```

Comment:
Not necessarily against this change, but any reason for it?

Reply:
to fix a conflict somewhere else

Reply:
Oh, then yea that'd be fine.